### ISSUE TITLE
[Feature Request] Add files in the same folder with similar name

### ISSUE THREAD

**Issue by CallumHoward** [lhc70000/iina#772]
**System and IINA version:**

- macOS 10.12.5
- IINA 0.0.11

**Expected behavior:**
Add option to "Add files in the same folder with similar name", which will find files with a common prefix or common substring and add them to the playlist, ordering by number. Similar to the behaviour of MPlayerX.
eg. for a set of files like
```
[yt]title_-_a_unique_title_[xvid]_S01E1.mp4
[yt]title_-_aother_unique_title_[xvid]_S01E02.mp4
[yt]title_-_a_third_title_[xvid]_S01E03.mp4
```
or
```
a_video_name_-_part1.mp4
a_video_name_-_part2.mp4
```
This would be useful for people like me, who are too lazy to sort things into folders, and just play videos from a big messy download folder.

**Comment by lhc70000** [comment]
The current auto load function should fit your requirement. Do you mean adding related files **only**?

**Comment by CallumHoward** [comment]
Yes, that is right, adding related files only, rather than every video file in the folder.
